AWSutils.sh

```diff
@@ -117,6 +126,12 @@ function InstallCLIv1 {
       rm -rf "${CHROOTMNT}/tmp/awscli-bundle.zip" \
          "${CHROOTMNT}/tmp/awscli-bundle" || \
         err_exit "Failed cleaning up install files"
+   elif [[ ${CLIV1SOURCE} == pip,* ]]
```

Comment:
What does this look like on the user side, far as the value they pass to `--cli-v1`?

Reply:
If someone wants to use pip to install, they would put a value of `pip,MODULENAME`  (where `MODULENAME` would normally be `awscli`. The thought here had been that, if AWS ever fixed the RPM or we _did_ decide to publish an RPM of the CLI would eventually add a block here that would allow/parse `rpm,RPMNAME` or `rpm,RPMURL`.